osd/cluster_support_suspended_ns.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Warning",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Cluster support suspended: non-authorised changes in openshift-* namespace detected",
+    "description": "Openshift Dedicated SRE team has noticed some non-supported changes that were made to the openshift-* namespace. This cluster will no longer be monitored and managed by the SRE team. Possible causes may include configuration changes to openshift-* namespaces, etc. If you wish to resume support for this cluster, please submit a support ticket.",
```

Comment:
Openshift Dedicated SRE team has noticed some non-supported changes that were made to the openshift-* namespace. This is a reserved namespace that should not be tampered with and as a consequence this cluster will no longer be monitored and managed by the SRE team. If you wish to resume support for this cluster, please submit a support ticket.